next.config.js

```diff
@@ -9,26 +9,14 @@ module.exports = {
     '@storybook/addon-a11y',
   ],
   async redirects() {
-    // Only make the permanent redirects really permanent in environments other
-    // than development, so we don't get permanent redirects on localhost which
-    // may conflict with other projects.
+    // Redirects to fix non-existing paths should go in `src/redirects.js`!!!
     const env = process.env.NEXT_PUBLIC_ENVIRONMENT;
     return [
       {
         source: '/',
         destination: '/dashboard',
```

Comment:
Is this kept on purpose for now? since we were goinf to change it to just the root for dashboard

Reply:
The new dashboard is going to be `/events` technically. (= The events tab on the dashboard)
But we have to keep this redirect here for `/` because `/` doesn't get handled by `[...params].js`, so our own redirect logic doesn't kick in.